.gitignore

```diff
@@ -7,3 +7,4 @@ _*
 .R*
 Meta
 doc
+opensourcecollaborators.csv
```

Comment:
Might I suggest that we simply prefix all temp/dev files with an underscore, and then simply add `_*.*` into our `.gitignore`?  Would that match your intention?